packages/rancher-monitoring/rancher-monitoring/generated-changes/patch/templates/prometheus/rules-1.14/alertmanager.rules.yaml.patch

```diff
@@ -8,8 +8,8 @@
  apiVersion: monitoring.coreos.com/v1
  kind: PrometheusRule
  metadata:
-@@ -212,4 +213,5 @@
-         {{ toYaml .Values.defaultRules.additionalRuleLabels | nindent 8 }}
+@@ -300,4 +301,5 @@
+         {{- end }}
        {{- end }}
  {{- end }}
 -{{- end }}
```

Comment:
I *think* this extra {{ end }} could cause helm template errors (and therefore helm install/upgrade errors)

Reply:
Added to fix some templating issues introduced in rebase, but there is an extra `if` above that requires the new `{{ end }}`